### PR TITLE
Fixed SSGI texture allocation with RG disabled

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
### Purpose of this PR

https://fogbugz.unity3d.com/f/cases/1293020/
SSGI works even if raytracing is not supported
removes if before RTHandles.Release because they are already done inside the function

---
### Testing status

tested the repro steps

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs [hunk 1]
```diff
@@ -850,24 +849,18 @@ void DestroyRenderTextures()
...
-                    RTHandles.Release(m_IndirectDiffuseHitPointBuffer);
-
                 m_RayTracingLightCluster.CleanupNonRenderGraphResources();
 
                 RTHandles.Release(m_FlagMaskTextureRT);
 
                 RaytracingManagerCleanupNonRenderGraphResources();
             }
+
+            RTHandles.Release(m_IndirectDiffuseBuffer0);
```
Comment (anisunity) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs:859]: The release function is safe even if the pointer is null?

**Comment by anisunity** [comment]
Did you run the DXR tests locally?

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs (hunk 1 above)
Comment (adrien-de-tocqueville) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs:859]: yep

**Comment by adrien-de-tocqueville** [comment]
> Did you run the DXR tests locally?

i don't support DXR locally, I can launch yamato but I guess it will be launched on the bugfix branch altogether

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
> > Did you run the DXR tests locally?
> 
> i don't support DXR locally, I can launch yamato but I guess it will be launched on the bugfix branch altogether

For this kind of thing, you should indeed run DXr test on yamato, I will do it

**Comment by sebastienlagarde** [comment]
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/hd%252Ffix-ssgi-no-rg/.yamato%252Fall-hdrp.yml%2523All_HDRP_trunk/4388893/job

**Comment by sebastienlagarde** [comment]
good for the test , merging

**Pull request merged**